Set up nextjs typescript project

motuncoded opened the pull request:
In this change, I set up a nextjs typescript project using the command npx create-next-app. I also made changes to the some certain files and remove the unnecessary files.

Closes #1

Commit:
Set up nextjs typescript project

In this change, I set up a nextjs typescript project using the command npx create-next-app. I also made changes to the some certain files and remmove the unnecessary files.

Commit:
Initial commit on orphan branch

Asylcreek commented on pages/index.tsx:
```diff
@@ -0,0 +1,14 @@
+import { Inter } from "next/font/google";
+
+const inter = Inter({ subsets: ["latin"] });
```
Move these into a separate `fonts` file

iamr0b0tx replied:
@motuncoded Can you make sure to resolve conversations when you have "fixed" them so that this page is cleaner to read and we can easily identify whatever is left unfixed

motuncoded replied:
Ok. Noted.

Asylcreek reviewed (changes requested):
Please setup Prettier in this app.

And is there a reason why you are using Tailwind?

Commit:
fix the changes

In this change, i uninstalled the tailwindcss configuration to be able to use the normal css styling and also install prettier and also moved the fonts to a file

Asylcreek commented on pages/fonts.ts:
This should not be inside the `pages` directory. Can you please go through NextJS docs to find out why?

motuncoded replied:
I will check upon that and correct that

Asylcreek reviewed (changes requested):
Also, I didn't ask you to remove Tailwind from your project, I only asked you why you were using it

Commit:
fix the setup change

In this change, I made changes by changing the directory for fonts file and install tailwindcss for general styling

iamr0b0tx commented:
General comment

@motuncoded  You can do better than "fix the changes" for commit titles. You can say something like, "moved xyz to pages directory".

Your commit history (titles) should read like a timeline or stories

motuncoded commented:
Ok I will take note of that
